mdb-realm/package.json

```diff
@@ -8,7 +8,7 @@
     "format": "prettier --write **/*.ts",
```

Comment:
I think the package.json version should be updated so that we can opt-in to this update.

Reply:
The version that is used by the workflow is the git tag/commit sha/branch name. The version in package.json is ignored in workflows.